app/models/user.rb

```diff
@@ -1,7 +1,12 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
   has_many :posts, foreign_key: 'author_id', dependent: :destroy
```

Comment:
As per project requirements, kindly make sure that user registration is confirmable by email. This can be achieved by uncommenting `:confirmable` in the user model.